packages/salesforcedx-vscode-core/src/commands/listMetadata.ts

```diff
@@ -35,9 +35,9 @@ export class ForceListMetadataExecutor extends SfdxCommandletExecutor<string> {
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   public build(data: {}): Command {
     const builder = new SfdxCommandBuilder()
-      .withArg('force:mdapi:listmetadata')
+      .withArg('org:list:metadata')
       .withFlag('-m', this.metadataType)
-      .withFlag('-u', this.defaultUsernameOrAlias)
+      .withFlag('-o', this.defaultUsernameOrAlias)
```

Comment:
@CristiCanizales should I remove this line, given the command does not give the user the ability to pick a different org?

Reply:
In the docs it says "Not required if the `target-org` configuration variable is already set.", so it would fail if the conf var is not set. Did you try without the line to check if it works?

Reply:
The describe manifest command does not use the target-org flag. Seems like it would not be needed here either.

Reply:
I'll test when doing the QA 👍

Reply:
I'll confirm the command will work without the `-o` flag